examples/nodejs/mongodb-examples/simple-read-aside/jest.config.ts

```diff
@@ -0,0 +1,14 @@
+import type {Config} from 'jest';
```

Comment:
There are some files that I would expect to see in this directory that aren't here in this PR.  Namely, at least:

- `package.json`
- `package-lock.json`
- `tsconfig.json`

Also, not necessarily required, but we generally want to have some `.eslint*` and `.prettier` files in here to enforce consistent code style.

Do you have a mechanism for allowing users to run without the files in my list above, though?